src/Repository/AdherentRepository.php

```diff
@@ -34,7 +34,7 @@ class AdherentRepository extends EntityRepository implements UserLoaderInterface
     const CITIZEN_INITIATIVE_RADIUS = 2;
     const CITIZEN_INITIATIVE_SUPERVISOR_RADIUS = 5;
 
-    public function count(): int
+    public function count(array $criteria = []): int
```

Comment:
I guess you did not want to commit that, no ?

Reply:
After this doctrine update, all `count()` methods in our repositories ovverides the new `Doctrine\ORM\EntityRepository::count(array $criteria)` method and should be compatible with it.

Symfony throw a `ContextErrorException` :
```
[Symfony\Component\Debug\Exception\ContextErrorException]                                                                                                  
Warning: Declaration of AppBundle\Repository\AdherentRepository::count(): int should be compatible with Doctrine\ORM\EntityRepository::count(array $criteria)
```

I think we can in another PR, change the `count()` name method in our repositories.

Reply:
Ok, bad news :O
And ok with your proposal

Reply:
I prefer to rename this method -> `countElements`. Because AdherentRepository is not the only one to have this issue. 
`EventRepository:count(bool $onlyPublished = true)`